### PR TITLE
Remove industries from peru rankings on PERU-152

### PR DESCRIPTION
**Pull request by makmanalp**
screen shot 2016-04-19 at 5 36 39 pm

### CODE REVIEW

File: app/mixins/table-map.js [hunk 1]
```diff
@@ -150,14 +150,21 @@ export default Ember.Mixin.create({
...
+       { key: 'name' },
+       { key: 'year' },
+       { key: 'eci' },
+       { key: 'gdp_pc_real' },
+       { key: 'gdp_real' },
+       { key: 'population' }
+      ];
+
+    if(this.get('featureToggle.showIndustries')) {
+      columns.splice(3, 0, {key: 'industry_eci'});
```
Comment (QuinnLee) [inline app/mixins/table-map.js]: smells like a python-ism 👍

Comment (makmanalp) [inline app/mixins/table-map.js]: Hahaha, glad to know you still are looking at the PRs :P The splice feels a bit weird but I couldn't really figure out how to make that column conditionally appear and also have it show up in the right spot and not just the end of all the columns.

Comment (QuinnLee) [inline app/mixins/table-map.js]: i think this is fine, but 4 space tabs?!

Comment (makmanalp) [inline app/mixins/table-map.js]: Oh shit that's my vim, good call!

**Pull request merged**